### PR TITLE
Indicate to user to specify hostname

### PR DESCRIPTION
**Pull request by morenoh149**
The deployment docs suggest using the hostname `dokku.me`. Instead put a placeholder prompting the user to specify the correct hostname.

[ci skip]

**Note:** If this PR is just doc changes, please put [ci skip] in the body that way tests do not run.

### CODE REVIEW

**Comment by josegonzalez** [comment]
I would rather we put a note at the top of the page stating that `dokku.me` is a placeholder for your domain name. We use `dokku.me` as that pretty much everywhere in our docs for standardization - and also because it makes it so if you setup the vagrant vm locally, the instructions just work.

File: docs/deployment/application-deployment.md [hunk 1]
```diff
@@ -55,14 +55,16 @@ Now you can deploy the `ruby-rails-sample` app to your Dokku server. All you hav
...
+git remote add dokku dokku@<your hostname>:ruby-rails-sample
 git push dokku master
 ```
 
 > Note: Some tools may not support the short-upstream syntax referenced above, and you may need to prefix
-> the upstream with the scheme `ssh://` like so: `ssh://dokku@dokku.me:ruby-rails-sample`
+> the upstream with the scheme `ssh://` like so: `ssh://dokku@<your hostname>:ruby-rails-sample`
 > Please see the [Git](https://git-scm.com/docs/git-clone#_git_urls_a_id_urls_a) documentation for more details.
 
+> Note: Your private key needs to be registered with ssh-agent. If you get a permission denied error when pushing you can register your private key by running `ssh-add -k ~/<your private key>`.
```
Comment (josegonzalez) [inline docs/deployment/application-deployment.md]: Add a note that this is to be done locally, not on the server.

**Comment by morenoh149** [comment]
squashed and updated

**Comment by josegonzalez** [comment]
Great work! Thanks for the pull request :)

**Pull request merged**